### PR TITLE
Fix eslint in alert component

### PR DESCRIPTION
**Pull request by emersonlaurentino**
#### What is the purpose of this pull request?

#trivial Fix eslint errors #1082

#### What problem is this solving?

Fix eslint in alert component

#### How should this be manually tested?

`yarn lint`

#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/Alert/index.test.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import Alert from './index'
 
 // Skipping those tests because getByText is getting the div inside the button and not the button element itself
 // I've tried to fix it, but I couldn't manage to. So I let me fellows future developers in this endeavour
-describe.skip('Alert', () => {
+describe('Alert', () => {
   describe('action button', () => {
     it('should be displayed if props are passed', () => {
       const label = 'Action'
```
Comment (klzns) [inline react/components/Alert/index.test.js]: The tests are not passing are they?

File: react/components/Alert/index.js [hunk 2]
```diff
@@ -92,6 +92,8 @@ class Alert extends Component {
             title="Close"
             className={`vtex-alert__close-icon pointer pv2 c-on-base ph3 ${innerVerticalPadding}`}
             onClick={onClose}
+            onKeyPress={onClose}
```
Comment (lucasecdb) [inline react/components/Alert/index.js]: shouldn't this be checking for the type of key pressed? like an 'enter' or 'space' 🤔

Comment (lucasecdb) [reply react/components/Alert/index.js]: imagine you tabbing into this "button" and pressing tab to focus the next element and the alert closing when you didn't expect it to

File: react/components/Alert/index.test.js (hunk 1 above)
Comment (emersonlaurentino) [reply react/components/Alert/index.test.js]: before they were not passed, now they are passing.

File: react/components/Alert/index.js (hunk 2 above)
Comment (emersonlaurentino) [reply react/components/Alert/index.js]: It really must be done!

Comment (emersonlaurentino) [reply react/components/Alert/index.js]: fixed @lucasecdb

**Pull request merged**